src/Illuminate/Support/Arr.php

```diff
@@ -342,6 +342,38 @@ public static function has($array, $keys)
         return true;
     }
 
+    /**
+     * Determine if any of the keys exist in an array using "dot" notation.
+     *
+     * @param  \ArrayAccess|array  $array
+     * @param  string|array  $keys
+     * @return bool
+     */
+    public static function hasAny($array, $keys)
+    {
+        if (is_null($keys)) {
+            return false;
+        }
+
+        $keys = (array) $keys;
+
+        if (! $array) {
+            return false;
+        }
+
+        if ($keys === []) {
```

Comment:
This check is actually redundant, since `! []` evaluates to `false`, and so would have been caught above.